### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by justinr1234**

### CODE REVIEW

File: imports/api/helpers/both/publication-names.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import { publicationNames as examplePublicationNames } from 'meteor/justinr1234:example';
-
-export const publicationNames = {
-  ...examplePublicationNames,
-};
+// export const publicationNames = {
```
Comment (zerkz) [inline imports/api/helpers/both/publication-names.js]: Not sure what kind of ship you're running here, but usually commented out code is a no go.

File: packages/justinr1234-lib/imports/startup/both/debug.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Meteor } from 'meteor/meteor';
+import { Tracker } from 'meteor/tracker';
+import { ReactiveVar } from 'meteor/reactive-var';
+import { SimpleSchema } from 'meteor/aldeed:simple-schema';
+
+export const DEBUG_ENABLED = new ReactiveVar(false);
+
+if (Meteor.isClient) {
+  Tracker.autorun(function debugAutorun() {
+    if (!!Meteor._localStorage.getItem('debug')) {
```
Comment (zerkz) [inline packages/justinr1234-lib/imports/startup/both/debug.js]: I take it that this is set automatically if the meteor server was started in debug?

File: imports/api/helpers/both/publication-names.js (hunk 1 above)
Comment (justinr1234) [inline imports/api/helpers/both/publication-names.js]: Leaving as an example of what could go here for now

File: packages/justinr1234-lib/imports/startup/both/debug.js (hunk 2 above)
Comment (justinr1234) [inline packages/justinr1234-lib/imports/startup/both/debug.js]: This is set manually in your browser if you want to enable debug

**Pull request merged**